js/initcontext_test.go

```diff
@@ -241,7 +241,7 @@ func TestInitContextOpen(t *testing.T) {
 		{[]byte("hello world!"), "ascii", 12},
 		{[]byte("?((¯°·._.• ţ€$ţɨɲǥ µɲɨȼ๏ď€ΣSЫ ɨɲ Ќ6 •._.·°¯))؟•"), "utf", 47},
 		{[]byte{0o44, 226, 130, 172}, "utf-8", 2}, // $€
-		//{[]byte{00, 36, 32, 127}, "utf-16", 2},   // $€
+		// {[]byte{00, 36, 32, 127}, "utf-16", 2},   // $€
```

Comment:
Why do we want to keep it?
```suggestion
```

Reply:
I don't want to remove code that isn't absolutely dead in a lint fixing pass. 

Maybe one day we will make this work :shrug: